src/arch/InputHandler/InputHandler_DirectInput.cpp

```diff
@@ -601,11 +580,8 @@ InputHandler_DInput::UpdateBuffered(
 						else if (in.ofs == DIMOFS_BUTTON2)
 							mouseInput = MOUSE_MIDDLE;
 						else
-							LOG->MapLog("unknown input",
-										"Mouse '%s' is returning an unknown "
-										"mouse offset [button], %i",
-										device.m_sName.c_str(),
-										in.ofs);
+							Locator::getLogger()->trace("Mouse '{}' is returning an unknown mouse offset [button], {}",
```

Comment:
We want to be careful converting MapLogs to traces due to their possibly high traffic usage. 
In this specific case, it might be better to make this a warn. If it crops up as an issue later, we could remove this log entirely or find a new solution.

Reply:
Fixed.